### DISCUSSION TITLE
Editing the titles bars of the status screens.

### DISCUSSION THREAD

**Discussion by erenaw** [TownyAdvanced/Towny#7495] [Q&A] [answered]
![2024-07-06_12 25 17](https://github.com/TownyAdvanced/Towny/assets/96159772/9e72afb7-f8db-422e-be3f-3778994507cf)
.oOo. Where do I need to enter to edit the part that says: I looked at en-US chatconfig and everywhere else.

**Comment by LlmDl** [comment, marked as answer]
That part is not editable.

**Reply by erenaw** [reply]
![2024-07-06_15 06 15](https://github.com/TownyAdvanced/Towny/assets/96159772/3c9900bb-7835-4948-b737-fff913d1ee02)
How did these men change it?

**Reply by LlmDl** [reply]
Dunno, probably by forking the plugin.
